### ISSUE TITLE
Improvise Map

### ISSUE THREAD

**Issue by Pramad712** [Pramad712/Thornton-Shortest-Paths#3]
- [ ] High Priority: Main Office Building
   - As of right now it is an awkwardly small rectangle. Plus the hallway is missing and the door is in the middle, not the corner.
   - There are lots of rooms there, such as the counselors' offices, the main office, and the library.
- [ ] Medium Priority + UI Improvement: Two buttons to switch from upper-floor view and lower-floor view.
   - This may help with the placement of the library in the new map
- [ ] Add 2 extra points for each classroom: Entrance & Exit
   - For some classrooms they are different.
   - This is definitely not as high of a priority, since most people should be able to find both of them easily being very close to each other. However, the most optimal route may change a bit.
- [ ] Lunch tables & Shade
   - Also low priority, since this would only be useful for lunch purposes. Classrooms are the main focus of this project.
- [ ] Update the executables once at least the first or second task is complete.
